lib/ddtrace.rb

```diff
@@ -25,8 +26,14 @@ module Datadog
   def self.tracer
     @tracer
   end
+
+  def self.registry
+    @registry
+  end
 end
 
+require 'ddtrace/monkey'
```

Comment:
there is a reason why it has been moved here and not at the top of the file?

Reply:
Yes. the problem is, `Monkey` is loading all integrations, and during their load time they're registering themselves in the `Datadog.registry`. So by the time `monkey` is required, we need `Datadog.registry` defined. Hence the move.

Once we have time I'd like to improve our startup process, so we rely less on ordering etc.

Reply:
yes after reading the rest of the PR I got that. Can you add a comment saying ^^? just in case someone else will work in the startup process in a few months.

Reply:
OK!